README.md

```diff
@@ -28,7 +28,7 @@ The OpenStack API's share API idioms like the used AuthN/AuthZ
 OpenStack services.
 
 Entering general IAM (Identity and Access Management), Keycloak has its own set of
-API endpoints and authentication flows.  
```

Comment:
Just curious: Why remove the `  ` at the line endings at some places?

I added them on purpose, actually, to have a break/separation that is bigger than none at all, but also not as big as a whole new paragraph.

If these non-paragraph newlines should go, maybe they should be replaced with a paragraph, instead, though.

Reply:
That was a mistake. My editor removed trailing whitespace automatically and it was not obvious these were intended to be there (some people leave trailing spaces without intention). Fixed!